### ISSUE TITLE
Beakerx TableDisplay breaks bqplot GridHeatMap

### ISSUE THREAD

**Issue by DougRzz** [twosigma/beakerx#6803]
Jupyter Notebook v5.3
OS: Windows 10
Beakerx v0.12.1
BQPlot v0.10.2
--------------------------------------
Hi,   I really like the BeakerX table.  Very nice job.    However, I'm having issues with TableDisplay when using BQplot heatmaps.  After I create a TableDisplay widget, the BQplot heatmap or gridheatmap goes blank.  

Here is an example:

```python
from __future__ import print_function
import numpy as np
from bqplot import *

np.random.seed(0)
data = np.random.randn(10, 10)

col_sc = ColorScale()
grid_map = GridHeatMap(color=data, scales={'color': col_sc})

fig = Figure(marks=[grid_map], padding_y=0.0)
fig
```

```python
import beakerx as bx
import pandas as pd
bx.TableDisplay(pd.DataFrame(index = [0],columns= ['blank']))
```

```python
fig
```

**Comment by scottdraves** [comment]
cool thanks for the report, we'll look into it.

**Comment by DougRzz** [comment]
Hi,   I love your new table.  However, this issue with bqplot doesn't seem to be fixed on my system.   

Jupyter Notebook v5.3
OS: Windows 10
Beakerx v0.15.2
BQPlot v0.10.5
ipywidgets v7.2.1

**Comment by scottdraves** [comment]
thanks!
It's fixed on master, should be released in about 2 weeks in 0.16.0.
